spec/spec_helper.rb

```diff
@@ -11,6 +11,13 @@ class Undef
   def inspect; 'undef'; end
 end
 
+# Workaround for slow rspec-puppet-facts
+def supported_os_opts
```

Comment:
The name threw me off here. Maybe a better name would be ```common_supported_os_opts``` with a note describing that this is for quick testing of a single OS.